rosdoc2/verbs/build/builders/doxygen_builder.py

```diff
@@ -79,6 +79,17 @@ def __init__(self, builder_name, builder_entry_dictionary, build_context):
 
         assert self.builder_type == 'doxygen'
 
+        # If the build type is not `ament_cmake/cmake`, there is no reason
+        # to create a doxygen builder.
+        if (
+            self.build_context.build_type not in
+            ('ament_cmake', 'cmake') and not self.build_context.always_run_doxygen
+        ):
+            logger.info(
+                f"The package build type is not 'ament_cmake' or 'cmake', hence the"
+                f"'{self.builder_type}' builder was not configured")
```

Comment:
```suggestion
            logger.debug(
                f"The package build type is not 'ament_cmake' or 'cmake', hence the "
                f"'{self.builder_type}' builder was not configured")
```

Reply:
Addressed in https://github.com/ros-infrastructure/rosdoc2/pull/28/commits/7d0b4fd4dff3ca0c92f8364ab39b47625f944dbf